redash/query_runner/treasuredata.py

```diff
@@ -68,7 +68,7 @@ def get_schema(self, get_stats=False):
         schema = {}
         if self.configuration.get("get_schema", False):
             try:
-                with tdclient.Client(self.configuration.get("apikey")) as client:
+                with tdclient.Client(self.configuration.get("apikey"),endpoint=self.configuration.get("endpoint")) as client:
```

Comment:
If `endpoint` is`None` it will use the default value?

Reply:
Yes, default parameter(https://api.treasuredata.com) is used